srcSpecs/demo-spec.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape');var file = require('../src/demo.js');test('sum function', function(t) { t.plan(2); 
```

Comment:
We probably don't wan't to have the demo specs in our development branch so this file should be removed.